modules/scheduler/scheduler/src/main/resources/webapp/js/controllers.js

```diff
@@ -240,6 +240,14 @@
                 job.endDate = $scope.parseToDateTime(job.endDate);
             }
 
+            if (job.startDate && job.endDate) {
+                if (job.startDate > job.endDate) {
+                    motechAlert("scheduler.error.incorrectJobDate", "scheduler.error");
+                    unblockUI();
```

Comment:
UI shouldn't be blocked at this point so the call of the unblockUI function is not needed.